app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Users::RegistrationsController < Devise::RegistrationsController
+  respond_to :json
+
+  private
+
+  def respond_with(resource, _opts = {})
+    register_success && return if resource.persisted?
+
+    register_failed
+  end
+
+  def register_success
+    render json: { message: 'Signed up sucessfully.' }
+  end
+
+  def register_failed
+    render json: { message: 'Something went wrong.' }
```

Comment:
I'm having issues with registration, and as we are not passing through errors, I'm not able to pinpoint the problem. 
![image](https://user-images.githubusercontent.com/59420634/163429323-fe744e5a-e3f9-4118-8bbd-c7ebf7dd9eb9.png)
## 
I used the same parameters on console and registration worked there.
## 
![image](https://user-images.githubusercontent.com/59420634/163429553-085a2431-70e7-46e1-9350-4d18445d3c08.png)


As this is not required at this stage, we can work on it later, but let me know if I'm missing anything, or if there is anything we can do to improve this.